src/ResourceManager/Websites/Commands.Websites/help/Enter-AzureRmWebAppContainerPSSession.md

```diff
@@ -0,0 +1,133 @@
+---
+external help file: Microsoft.Azure.Commands.Websites.dll-Help.xml
+Module Name: AzureRM.Websites
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.websites/?view=azurermps-6.8.1
+schema: 2.0.0
+---
+
+# Enter-AzureRmWebAppContainerPSSession
+
+## SYNOPSIS
+
+Opens a remote PowerShell session into the windows container specified in a given site or slot and given resource group
+
+
+## SYNTAX
+
+### S1
+```
+Enter-AzureRmWebAppContainerPSSession [[-SlotName] <String>] [-ResourceGroupName] <String> [-Name] <String>
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### S2
+```
+Enter-AzureRmWebAppContainerPSSession [-WebApp] <PSSite> [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+opens a remote PowerShell session into the windows container specified in a given site or slot and given resource group
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Enter-AzureRmWebAppContainerPSSession -ResourceGroupName "Default-Web-WestUS" -Name "ContosoASP"
+```
+
+This command opens a remote PowerShell session into the windows container app ContosoASP
+## PARAMETERS
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: IAzureContextContainer
```

Comment:
Please regenerate help using the `UseFullTypeName` parameter. See: https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module

Reply:
done